src/quo2/foundations/colors.cljs

```diff
@@ -296,10 +296,6 @@
    (let [theme (or override-theme (theme/get-theme))]
      (if (= theme :light) light dark))))
 
-(defn dark?
```

Comment:
🎯 remove this method. Imo it is odd to mix the colors and theming functionality/mechanisms like this. We should keep them separate in implementation and uses.